core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -756,9 +761,9 @@ public String toString(@Nullable AbstractBlockChain chain, @Nullable CharSequenc
         s.append('\n');
         if (updatedAt != null)
             s.append(indent).append("updated: ").append(Utils.dateTimeFormat(updatedAt)).append('\n');
-        if (version != 1)
+        if (version != CURRENT_VERSION)
             s.append(indent).append("version ").append(version).append('\n');
```

Comment:
The goal here is to append version information if the version is other than 1.  But if CURRENT_VERSION is changed to `2`, then it would add this information for all versions except `2`

Reply:
I see. This was my thinking, but I assumed the goal is to append information if the version isn't the same as the current version.